packages/react-components/react-menu/library/src/components/MenuSplitGroup/renderMenuSplitGroup.tsx

```diff
@@ -1,13 +1,18 @@
 /** @jsxRuntime automatic */
 /** @jsxImportSource @fluentui/react-jsx-runtime */
 import { assertSlots } from '@fluentui/react-utilities';
-import type { MenuSplitGroupState, MenuSplitGroupSlots } from './MenuSplitGroup.types';
+import type { MenuSplitGroupState, MenuSplitGroupSlots, MenuSplitGroupContextValues } from './MenuSplitGroup.types';
+import { MenuSplitGroupContextProvider } from '../../contexts/menuSplitGroupContext';
 
 /**
  * Render the final JSX of MenuSplitGroup
  */
-export const renderMenuSplitGroup_unstable = (state: MenuSplitGroupState) => {
+export const renderMenuSplitGroup_unstable = (state: MenuSplitGroupState, contexts: MenuSplitGroupContextValues) => {
```

Comment:
BC?

Reply:
good point - changed to an optional arg and uses a default value for the context if not present